src/components/settings/SettingsSSOTab/index.jsx

```diff
@@ -26,6 +29,11 @@ const SettingsSSOTab = ({ enterpriseId, setHasSSOConfig }) => {
         validConfigExists = true;
       }
     });
+    if (existingConfigs?.length < 1 || !existingConfigs) {
```

Comment:
nit: Check for falsiness first
```suggestion
    if (!existingConfigs || existingConfigs?.length < 1) {
```